remoto/tests/backends/test_backends.py

```diff
@@ -71,6 +71,15 @@ def test_fallback_interpreter(self, monkeypatch, capsys):
             pass
         assert remote_fake_module.python_executable is not None
 
+    def test_remote_python(self):
+        import sys
+        remote_python = sys.executable
```

Comment:
I understand this is pointing to the existing Python path, but could you add another test that uses an arbitrary path as well? I know it _should_ fail because the executable wouldn't exist, but just want some validation that what you pass in in `import_module` sticks all the way to the `JsonModuleExecute` class

Reply:
Added `test_wrong_python_executable`.